src/component-systems/DeveloperSystem.js

```diff
@@ -2,12 +2,15 @@ import * as THREE from 'three';
 import System from '../core/System';
 import { COLLIDER_CURSOR_MAP } from './RapierPhysicsSystem';
 
+// TODO - is this item still needed?
```

Comment:
@michaelthatsit - quick question for you

Reply:
This whole system can be thrown out. it was for the initial demo and we will be removing this from the framework and building it on the platform.